metrique/tests/renames.rs

```diff
@@ -56,3 +65,18 @@ fn metrics_renames_work() {
     // For sub metrics which _don't_ set a rename, they get rename transitively from the pattern
     assert_eq!(entry.metrics["sub-field-a"], 4);
 }
+
+#[test]
+fn explicit_prefix_metrics() {
+    let metrics = ExplicitPrefixedMetrics {
+        a: 100,
+        local_rename: "abcd",
+    };
```

Comment:
some additional tests that are probably useful, at least to capture the current behavior
```
#[metrics(explicit_prefix = "API@")]
struct Outer {
    #[metrics(flatten, prefix = "inner_")]  // Mix of explicit + inflectable
    inner: Inner,
}
```

```
#[metrics(explicit_prefix = "API@")]
struct Outer {
    #[metrics(flatten, explicit_prefix = "Inner:")]
    inner: Inner,  // Does this become "API@Inner:field"?
}
```